akka-http-core/src/main/scala/akka/http/impl/engine/ws/Handshake.scala

```diff
@@ -22,7 +28,8 @@ import scala.reflect.ClassTag
 private[http] object Handshake {
   val CurrentWebsocketVersion = 13
```

Comment:
If there's not yet a test for it yet we should add one and see what happens. The autobahn test suite doesn't really cover the handshake.